components/NuxtLinkInline.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <NuxtLink class="link-inline" :to="nextPath" :append="nextPathAppend">
+    <slot></slot>
+  </NuxtLink>
+</template>
+
+<script>
+export default {
+  props: {
+    nextPath: {
+      type: String,
+      required: true,
+      default: null,
+    },
+    nextPathAppend: {
+      type: Boolean,
+      required: false,
+      default: true,
+    },
+  },
+}
+</script>
+
+<style scoped lang="scss">
+.link-inline {
```

Comment:
the way these components are architected feels weird:

- components/NuxtLinkInline.vue
- components/NuxtLinkPrimary.vue

They just wrap the native NuxtLink and only add styling that is tightly coupled with the component. The only difference between the components is the class that gets applied.

This can be done in a simpler way:

1. define `.link-inline` and `.link-primary` styles globally
2. apply these classes to the native NuxtLink component
3. done :) (no additional components necessary)

Reply:
perfect, i will do that :)